src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetSharedLiveMedia/SharedLiveMediaItem/DeleteSharedLiveMediaUploadButton/index.tsx

```diff
@@ -3,31 +3,32 @@ import React from 'react';
 import { defineMessages, useIntl } from 'react-intl';
 
 import { BinSVG } from 'components/SVGIcons/BinSVG';
-import { useDeleteUploadModal } from 'data/stores/useDeleteUploadModal';
+import { useDeleteSharedLiveMediaUploadModal } from 'data/stores/useDeleteSharedLiveMediaUploadModal';
 import { SharedLiveMedia } from 'types/tracks';
 
 const messages = defineMessages({
   buttonLabel: {
     defaultMessage: 'Click on this button to delete the media.',
     description: 'The label of the button to delete a media.',
-    id: 'components.DeleteUploadButton.buttonLabel',
+    id: 'components.DeleteSharedLiveMediaUploadButton.buttonLabel',
   },
 });
 
-interface DeleteUploadButtonProps {
+interface DeleteSharedLiveMediaUploadButtonProps {
   sharedLiveMedia: SharedLiveMedia;
 }
 
-export const DeleteUploadButton = ({
+export const DeleteSharedLiveMediaUploadButton = ({
```

Comment:
I don't understand this rename. is it a `deletion` or `upload` button ? It's confusing IMO

Reply:
If I may, previous naming was also about a DeleteUpload, I guess it is more a button to delete an uploaded shared media.
Since there is no shared media not uploaded, as far as I know, we can probably rename this `DeleteSharedMediaButton`

Reply:
done